tests/conftest.py

```diff
@@ -62,6 +63,26 @@ def increment_cache_key_prefix():
     cache.key_prefix = "t{0}".format(key_prefix_counter)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def block_unmocked_requests():
```

Comment:
That doesn't work, for the reasons mention in the code comment below.
Happy to reword the comment if there's a way of phrasing it that makes it more obvious?